packages/teams-js/src/internal/mediaUtil.ts

```diff
@@ -52,6 +55,38 @@ export function decodeAttachment(attachment: media.MediaChunk, mimeType: string)
   return assemble;
 }
 
+/**
+ * @hidden
+ * Function returns null if the media call is supported on current mobile version, else SdkError.
+ *
+ * @internal
+ */
+export function isMediaCallSupportedOnMobile(mediaInputs: media.MediaInputs): SdkError {
```

Comment:
Why are we returning an error? Isn't it standard practice to throw an Error when it is not supported?

Reply:
To get unblocked, tabling this issue. Will create a work item.

Reply:
Same thing: please link whatever you open to track this! Might be worth combining with the similar issue from internalAPI.ts?

Reply:
As discussed offline, I am creating a bug item and linking it in our Teams channel.